src/app/shared/utilities/parse-version.ts

```diff
@@ -0,0 +1,9 @@
+export function parseVersion(data) {
+  // NOTE: Based on current Google Play Store page pattern
+  const playstorePattern = /<span class="htlgb">([0-9]*[.][0-9]*[.][0-9]*)<\/span>/g
+  const versionPattern = /([0-9]*[.][0-9]*[.][0-9]*)/
```

Comment:
alpha versions?

Reply:
Forgot about it, will make the changes.

Reply:
ohh but alpha will not be visible on the public domain. So i guess we can only do for production releases

Reply:
Oh okay